jvm-packages/xgboost4j-example/src/main/scala/ml/dmlc/xgboost4j/scala/example/BoostFromPrediction.scala

```diff
@@ -48,6 +49,6 @@ object BoostFromPrediction {
     testMat.setBaseMargin(testPred)
 
     System.out.println("result of running from initial prediction")
-    val booster2 = XGBoost.train(trainMat, params.toMap, 1, watches.toMap, null, null)
+    XGBoost.train(trainMat, params.toMap, 1, watches.toMap, null, null)
```

Comment:
What is this example doing if it's not showing the result?

Reply:
Guess it's just trying to verify if the training is ok.